src/content/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing.mdx

```diff
@@ -78,7 +78,7 @@ To learn how billable users are calculated, see [User count calculation](/docs/a
 A New Relic organization can have one of several types of plans that govern how billing scheduling is handled. Here are the main options:
 
 * **Pay-as-you-go**: With this option, billing happens at the end of each calendar month. There are no payment commitments past the current month and an organization can cancel at any time. An organization's first end-of-month bill includes billable usage starting from when their credit card information was input into New Relic.
-* **Contracts**: Contract options include our **annual pool of funds** option, and other options. With contracts, an estimate of long-term expected usage (ingested data and billable users) is created and there's an agreed upon payment commitment. If an organization's usage exceeds that commitment, the excess usage is billed at the previously agreed-upon rates. These are available primarily to Pro and Enterprise edition organizations. 
+* **Contracts**: Contract options include our Volume Plan, Savings Plan, and other options. With contracts, an estimate of long-term expected usage (ingested data and billable users) is created and there's an agreed upon payment commitment. If an organization's usage exceeds that commitment, the excess usage is billed at the previously agreed-upon rates. These are available primarily to Pro and Enterprise edition organizations. 
```

Comment:
I assume these have to be capitalized for legal reasons? Feels weird tho

Reply:
yeah they capitalize a whole lot of stuff